Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -317,6 +317,21 @@ class SourceGenerator {
         )
     }
 
+    private func expandPattern(_ pattern: String) -> [Path] {
+        let filePaths = listFilePaths(pattern: pattern)
+        let urls = filePaths.map { Path($0) }
+
+        return urls
+    }
+
+    private func listFilePaths(pattern: String) -> [String] {
```

Comment:
This class already has too many functions (needs a refactor). For now can we just combine these two functions into one? Our basic currency type is always Path